packages/squiggle-lang/__tests__/Distributions/Scale_test.res

```diff
@@ -0,0 +1,19 @@
+/*
+This test case comes via Nuño https://github.com/quantified-uncertainty/squiggle/issues/433
+
+
+*/
+
+open Jest
+open Expect
+open TestHelpers
+
+describe("", () => {
```

Comment:
? Empty names?

Reply:
I was confused for a second because this isn't the case in my emacs buffer--- I actually forgot to `:w` it turns out. sorry!